rootfs/api/urls.py

```diff
@@ -29,22 +29,6 @@
         views.ReleaseViewSet.as_view({'post': 'rollback'})),
     url(r"^apps/(?P<id>{})/releases/?".format(settings.APP_URL_REGEX),
         views.ReleaseViewSet.as_view({'get': 'list'})),
-    # restart containers
```

Comment:
We also have https://github.com/deis/controller/issues/363 - should that be closed down in favour of the issue you created?